index.html

```diff
@@ -25,61 +27,111 @@
         </h3>
         <h3 class="text-primary font-mulish-bold text-[24px]">Trafalgar</h3>
       </div>
-      <nav>
-        <img src="/assets/images/List.svg" class="cursor-pointer" />
+      <nav class="flex lg:justify-center lg:gap-[40px]">
```

Comment:
suggestion: for this cases that we have a tailwind class use `gap-10` instead of `gap-[40px]`